cmd/scheduler-estimator/app/options/options.go

```diff
@@ -64,6 +74,10 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.ClusterName, "cluster-name", o.ClusterName, "Name of member cluster that the estimator serves for.")
 	fs.StringVar(&o.BindAddress, "bind-address", defaultBindAddress, "The IP address on which to listen for the --secure-port port.")
 	fs.IntVar(&o.ServerPort, "server-port", defaultServerPort, "The secure port on which to serve gRPC.")
+	fs.StringVar(&o.CertFile, "grpc-auth-cert-file", "", "SSL certification file used for grpc SSL/TLS connections.")
+	fs.StringVar(&o.KeyFile, "grpc-auth-key-file", "", "SSL key file used for grpc SSL/TLS connections.")
+	fs.BoolVar(&o.ClientCertAuth, "client-cert-auth", false, "When this is set, server will check all incoming HTTPS requests for a client certificate signed by the trusted CA, requests that don’t supply a valid client certificate will fail. If authentication is enabled, the certificate provides credentials for the user name given by the Common Name field.")
```

Comment:
```suggestion
	fs.BoolVar(&o.ClientCertAuth, "client-cert-auth", true, "When this is set, server will check all incoming HTTPS requests for a client certificate signed by the trusted CA, requests that don’t supply a valid client certificate will fail. If authentication is enabled, the certificate provides credentials for the user name given by the Common Name field.")
```
Why not default to `true`?

Reply:
Mainly to ensure compatibility with previous ones. If default to `true`, then the `estimator` and `scheduler/deshceduler` will report an error if they are configured with no certificates like before.